gensim/topic_coherence/indirect_confirmation_measure.py

```diff
@@ -26,6 +26,23 @@
 
 logger = logging.getLogger(__name__)
 
+
```

Comment:
Please make valid latex by putting { and } in the formulas
 `m_sim(m, gamma)(W', W*) = s_sim(V_{m,gamma}(W'), V_{m,gamma}(W*))`